DCOS/dcos-testing.sh

```diff
@@ -829,6 +829,43 @@ run_dcos_autoscale_job() {
     return 0
 }
 
+run_fluentd_tests() {
+    echo "Checking if jumphost has all the required packages for winrm connection"
+    setup_remote_winrm_client || return 1
+    if [[ $WIN_PRIVATE_AGENT_COUNT -gt 0 ]]; then
+        echo "Looking for Windows private agents"
+        IPS=$($DIR/utils/dcos-node-addresses.py --operating-system 'windows' --role 'private') || return 1
+        for IP in $IPS; do
+            echo "Running Fluentd Pester tests on Windows private agent: $IP"
+            start_fluentd_tests "$IP" || return 1
+        done
+    fi
+    if [[ $WIN_PUBLIC_AGENT_COUNT -gt 0 ]]; then
+        echo "Looking for Windows public agents"
+        IPS=$($DIR/utils/dcos-node-addresses.py --operating-system 'windows' --role 'public') || return 1
+        for IP in $IPS; do
+            echo "Running Fluentd Pester tests on Windows public agent: $IP"
+            start_fluentd_tests "$IP" || return 1
+        done
+    fi
+}
+
+start_fluentd_tests() {
+    local AGENT_IP="$1"
+    WIN_REMOTE_CLONE_DIR="C:\\mesos-jenkins"
+    WIN_REMOTE_CMD="if (Test-Path -Path $WIN_REMOTE_CLONE_DIR) { Remove-Item -Force -Recurse -Path $WIN_REMOTE_CLONE_DIR }; git clone  https://github.com/capsali/mesos-jenkins $WIN_REMOTE_CLONE_DIR; pushd ${WIN_REMOTE_CLONE_DIR}\\DCOS\\fluentd-testing; \$results = Invoke-Pester -PassThru; \$FailedTestTally = \$results.FailedCount; if (\$FailedTestTally -eq 0) { write-host \"All tests passed successfully\"; exit 0} else { write-host \"Some tests failed. Number of failed tests is: \$FailedTestTally\"; exit 1}; popd"
```

Comment:
Remove hard-coded value https://github.com/capsali/mesos-jenkins
Add a variable above just like `WIN_REMOTE_CLONE_DIR` and name it `WIN_REMOTE_REPO_URL`. Please set it to https://github.com/Microsoft/mesos-jenkins

Reply:
I'm not a fan of this one-line PowerShell script. Add a script to execute the Pester tests to https://github.com/Microsoft/mesos-jenkins/tree/master/DCOS/fluentd-testing and you simply call it after you clone the repository.

Reply:
Also, use `Write-Output` instead of `Write-Host` since `Write-Host` doesn't write to standard output and `Write-Output` does.